tempesta_fw/http_parser.c

```diff
@@ -86,9 +87,30 @@ enum {
 static inline unsigned char *
 memchreol(const unsigned char *s, size_t n)
 {
+#ifdef __AVX2__
+    __m256 __avx_constants2 = __constants2;
```

Comment:
`__constants2` aren't defined in `tempesta_fw` tree, but in `./t/unit/dl-182/avx_routines.h` only.